test/integration-serial/serial_task_run_test.rb

```diff
@@ -28,4 +28,57 @@ def test_run_without_verify_result_fails_if_pod_was_not_created
       "Kubeclient::HttpError: Pod with same name exists"
     ], in_order: true)
   end
+
+  # We can't ensure that all the metrics are specific test when they are running in parallel
+  # so we run them serially
+  def test_failure_statsd_metric_emitted
+    deploy_task_template
+    task_runner = build_task_runner
+
+    result = false
+    metrics = capture_statsd_calls do
+      result = task_runner.run(run_params.merge(args: ["/not/a/command"]))
+    end
+
+    assert_task_run_failure(result)
+
+    metric_tags = metrics.detect { |m| m.tags.include? "namespace:#{@namespace}" }.tags
+    assert_includes metric_tags, "namespace:#{@namespace}"
+    assert_includes metric_tags, "context:#{KubeclientHelper::TEST_CONTEXT}"
+    assert_includes metric_tags, "status:failure"
+  end
+
+  def test_success_statsd_metric_emitted
+    deploy_task_template
+    task_runner = build_task_runner
+
+    result = false
+    metrics = capture_statsd_calls do
+      result = task_runner.run(run_params.merge(args: %w(ls)))
```

Comment:
similar to my comment above, can we run this with `verify_result: false` to speed it up? We don't really care how it succeeds, just that it does.